snap/hooks/install

```diff
@@ -33,10 +33,30 @@ for service in security-api-gateway security-secret-store core-command config-se
     fi
 done
 
-# also handle config-seed/device-virtual/application.properties
-if [ ! -f "${SNAP_DATA}/config/config-seed/res/properties/device-virtual/application.properties" ]; then
-    mkdir -p "${SNAP_DATA}/config/config-seed/res/properties/device-virtual"
-    cp "${SNAP}/config/config-seed/res/properties/device-virtual/application.properties" "${SNAP_DATA}/config/config-seed/res/properties/device-virtual/application.properties"
+# handle device-mqtt driver configuration
+cp ${SNAP}/config/device-mqtt/res/configuration-driver.toml "${SNAP_DATA}/config/device-mqtt/res/configuration-driver.toml"
+
+# handle device-random device profile
+cp ${SNAP}/config/device-random/res/device.random.yaml "${SNAP_DATA}/config/device-random/res/device.random.yaml"
+
+# also handle java services' application.properties
+for jsvc in device-virtual; do
```

Comment:
Since we're down to a single service, doesn't it make sense to get rid of this **for** loop, and just keep the existing logic in the master version?

Reply:
On the off chance that we need to add more java services, I'd rather leave it here.